cost-analyzer/templates/_helpers.tpl

```diff
@@ -398,31 +407,264 @@ The implied use case is {{ template "cost-analyzer.filterEnabled" .Values }}
+      value: {{ .Release.Namespace }}
+{{- end }}
+
+
+{{- define "aggregator.jaeger.sidecarContainerTemplate" }}
+- name: embedded-jaeger
+  securityContext:
+    {{- toYaml .Values.kubecostAggregator.jaeger.containerSecurityContext | nindent 4 }}
+  image: {{ .Values.kubecostAggregator.jaeger.image }}:{{ .Values.kubecostAggregator.jaeger.imageVersion }}
+{{- end }}
+
+
+{{- define "aggregator.cloudCost.containerTemplate" }}
+- name: cloud-cost
+  {{- if .Values.kubecostModel }}
+  {{- if .Values.kubecostAggregator.fullImageName }}
+  image: {{ .Values.kubecostAggregator.fullImageName }}
+  {{- else if .Values.kubecostModel.fullImageName }}
+  image: {{ .Values.kubecostModel.fullImageName }}
+  {{- else if .Values.imageVersion }}
+  image: {{ .Values.kubecostModel.image }}:{{ .Values.imageVersion }}
+  {{- else }}
+  image: {{ .Values.kubecostModel.image }}:prod-{{ $.Chart.AppVersion }}
+  {{ end }}
+  {{- else }}
+  image: gcr.io/kubecost1/cost-model:prod-{{ $.Chart.AppVersion }}
+  {{ end }}
+  readinessProbe:
+    httpGet:
+      path: /healthz
+      port: 9005
+    initialDelaySeconds: 10
+    periodSeconds: 5
+    failureThreshold: 200
+  imagePullPolicy: Always
+  args: ["cloud-cost"]
+  ports:
+    - name: tcp-api
+      containerPort: 9005
+      protocol: TCP
+  resources:
+    {{- toYaml .Values.kubecostAggregator.cloudCost.resources | nindent 4 }}
+  volumeMounts:
+  {{- if .Values.kubecostModel.federatedStorageConfigSecret }}
+    - name: federated-storage-config
+      mountPath: /var/configs/etl/federated
+      readOnly: true
+  {{- end }}
+  {{- if .Values.kubecostModel.etlBucketConfigSecret }}
+    - name: etl-bucket-config
+      mountPath: /var/configs/etl
+      readOnly: true
+  {{- end }}
+
+  {{- if .Values.kubecostProductConfigs }}
+  {{- if .Values.kubecostProductConfigs.cloudIntegrationSecret }}
+    - name: {{ .Values.kubecostProductConfigs.cloudIntegrationSecret }}
+      mountPath: /var/configs/cloud-integration
+  {{- else }}
+    # In this case, the cloud-integration is expected to come from the UI or
+    # from workload identity.
+    - name: cloud-integration
+      mountPath: /var/configs/cloud-integration
+  {{- end }}
+    # In this case, the cloud-integration is expected to come from the UI or
+    # from workload identity.
+    - name: cloud-integration
+      mountPath: /var/configs/cloud-integration
+  {{- end }}
+
+  env:
+    - name: CONFIG_PATH
+      value: /var/configs/
+    {{- if .Values.kubecostModel.etlBucketConfigSecret }}
+    - name: ETL_BUCKET_CONFIG
+      value: "/var/configs/etl/object-store.yaml"
+    {{- end}}
+    {{- if .Values.kubecostModel.federatedStorageConfigSecret }}
+    - name: FEDERATED_STORE_CONFIG
+      value: "/var/configs/etl/federated/federated-store.yaml"
+    - name: FEDERATED_CLUSTER
+      value: "true"
+    {{- end}}
+    - name: CLOUD_COST_REFRESH_RATE_HOURS
+      value: {{ .Values.kubecostAggregator.cloudCost.refreshRateHours | default  6 | quote }}
```

Comment:
```suggestion
      value: {{ .Values.kubecostAggregator.cloudCost.refreshRateHours | default 6 | quote }}
```

Reply:
Value not defined. Please add.

Reply:
In situations like this, do you want the value actually _defined_ (with a value) in values.yaml, or just listed in the values.yaml but commented out?

Reply:
Listed in a commented-out state at a minimum.